lib/iris/__init__.py

```diff
@@ -283,515 +282,23 @@ def context(self, **kwargs):
-    except ValueError:
-        raise iris.exceptions.ConstraintMismatchError("no cubes found")
-
-    return cube
-
-
-def load_cubes(uris, constraints=None, callback=None):
-    """Load exactly one Cube for each constraint.
-
-    For a full description of the arguments, please see the module
-    documentation for :mod:`iris`.
-
-    Parameters
-    ----------
-    uris :
-        One or more filenames/URIs, as a string or :class:`pathlib.PurePath`.
-        If supplying a URL, only OPeNDAP Data Sources are supported.
-    constraints : optional
-        One or more constraints.
-    callback : optional
-        A modifier/filter function.
-
-    Returns
-    -------
-    :class:`iris.cube.CubeList`
-        An :class:`iris.cube.CubeList`. Note that there is no inherent order
-        to this :class:`iris.cube.CubeList` and it should be treated as if it
-        were random.
-
-    """
-    # Merge the incoming cubes
-    collection = _load_collection(uris, constraints, callback).combined()
-
-    # Make sure we have exactly one merged cube per constraint
-    bad_pairs = [pair for pair in collection.pairs if len(pair) != 1]
-    if bad_pairs:
-        fmt = "   {} -> {} cubes"
-        bits = [fmt.format(pair.constraint, len(pair)) for pair in bad_pairs]
-        msg = "\n" + "\n".join(bits)
-        raise iris.exceptions.ConstraintMismatchError(msg)
-
-    return collection.cubes()
-
-
-def load_raw(uris, constraints=None, callback=None):
-    """Load non-merged cubes.
-
-    This function is provided for those occasions where the automatic
-    combination of cubes into higher-dimensional cubes is undesirable.
-    However, it is intended as a tool of last resort! If you experience
-    a problem with the automatic combination process then please raise
-    an issue with the Iris developers.
-
-    For a full description of the arguments, please see the module
-    documentation for :mod:`iris`.
-
-    Parameters
-    ----------
-    uris :
-        One or more filenames/URIs, as a string or :class:`pathlib.PurePath`.
-        If supplying a URL, only OPeNDAP Data Sources are supported.
-    constraints : optional
-        One or more constraints.
-    callback : optional
-        A modifier/filter function.
-
-    Returns
-    -------
-    :class:`iris.cube.CubeList`
-
-    """
-    from iris.fileformats.um._fast_load import _raw_structured_loading
+import iris.loading
+from iris.loading import (
+    CombineOptions,
+    LoadPolicy,
+    load,
+    load_cube,
+    load_cubes,
+    load_raw,
+)
 
-    with _raw_structured_loading():
-        return _load_collection(uris, constraints, callback).cubes()
+# NOTE: we define an independent local 'LOAD_POLICY' object here, just so that we can
+# ensure an entry in the API documentation output page.
 
+#: A control object containing the current file loading strategy options.
+LOAD_POLICY = iris.loading.LOAD_POLICY
 
-save = iris.io.save
+from iris.io import save
```

Comment:
Is there a reason for the import order for these save/load functions being so delayed? Would this not make more sense going next to `import iris.io`?

Reply:
No real reason, and I'm surprised that the isort and ruff-type checks didn't complain about it.
To me it does make sense that, if I were defining these things, I'd put them here in the file, because that's where they "used to live".  But now I'm importing them from somewhere else.

If you think it now makes more sense., I'll treat these like imports, "LOAD_POLICY" included, and move it all to the top.